Projekt dla ambitnych 1/Gruntfile.js

```diff
@@ -0,0 +1,68 @@
+module.exports = function(grunt) {
+
+  // Project configuration.
+  	grunt.initConfig({
+  		sass: {
+     		options: {
+        		sourceMap: true
+        	},
+        	dist: {
+        		files: {
+         			 'css/main.css': 'sass/main.sass'
+      	  		}
+    		}
+  	 	},
+
+  	 	imagemin: {
+    		dynamic: {
```

Comment:
`dynamic:` is a child of `imagemin:`. so you should add one more tab to it, to show relationship